protected/components/ModelSearch.php

```diff
@@ -356,9 +356,9 @@ public function retrieveResults()
     /**
      * Add a filter for active
      */
-    public function addActiveFilter($active_label )
+    public function addActiveFilter($active_label=null)
     {
-        $this->addSearchItem($active_label?:'active', array('type' => 'boolean'));
+        $this->addSearchItem($active_label == null?'active': $active_label, array('type' => 'boolean'));
```

Comment:
The line that was removed here is actually correct. The only issue was with the function parameter.

Reply:
fixed